File: PHP/tutorial/exp7_practical/time.php
<?php
    $time_now = time();
    echo "time() > " . time() . "<br>\n";
    echo "mktime(0, 0, 0, 1, 1, 2000) > ";
    echo mktime(0, 0, 0, 1, 1, 2000);
    echo "<br>\n";
    $sec  = $time_now%60;
    $min  = ($time_now/60)%60;
    $hour = ($time_now/3600)%24;
    echo "$hour:$min:$sec(GMT)<br>\n";
    $hour += 9;
    echo "$hour:$min:$sec(JST)<br>\n";
?>

File: Ruby/perfect_ruby/exp5/time.rb
p "TIME > " + Time.now.to_s

time = Time.new

p "ZONE > " + time.zone
p "UTC TIME > " + time.getutc.to_s

p "year > " + time.year.to_s
p "month > " + time.month.to_s
p "day > " + time.day.to_s
p "hour > " + time.hour.to_s
p "min > " + time.min.to_s
p "sec > " + time.sec.to_s
p "nsec > " + time.nsec.to_s

p "wday > " + time.wday.to_s
p "yday > " + time.yday.to_s
